scripts/generate-google3-docs/resource-reference/templates/compute_computenetworkfirewallpolicy.tmpl

```diff
@@ -0,0 +1,54 @@
+{{template "headercomment.tmpl" .}}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}{{ .Kind}}{% endblock %}
+{% block body %}
+
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Name</td>
+<td>Compute Engine</td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} Service Documentation</td>
+<td><a href="/compute/docs/">/compute/docs/</a></td>
+</tr>
+<tr>
+<td>{{"{{gcp_name_short}}"}} REST Resource Name</td>
+<td>v1.networkFirewallPolicies</td>
```

Comment:
I guess this should be `beta.networkFirewallPolicies` as it is talking to the Beta endpoint?

This is not a blocker as we can always adjust it whenever needed. And it is not consistently documented already.

Reply:
This will be fixed in the next PR for FirewallPolicyAssociation